content/en/docs/concepts/windows/user-guide.md

```diff
@@ -181,29 +183,37 @@ appropriate operating system.
 The `.spec.os.name` value has no effect on the scheduling of the Windows pods,
 so taints and tolerations and node selectors are still required
  to ensure that the Windows pods land onto appropriate Windows nodes.
+=======
+The scheduler does not use the value of `.spec.os.name` when assigning Pods to nodes.
+You should use normal Kubernetes mechanisms for
+[assigning pods to nodes](/docs/concepts/scheduling-eviction/assign-pod-node/)
+to ensure that the control plane for your cluster places pods onto nodes that are running the
+appropriate operating system.
+{{< /note >}}
+>>>>>>> Moving windows containers user guide out of /setup/production-environment/:content/en/docs/concepts/windows/user-guide.md
```

Comment:
This looks like a broken merge that needs tidying.

Reply:
Fixed, I'll scan the rest of the PR again now too.